common4j/src/main/com/microsoft/identity/common/java/BuildValues.java

```diff
@@ -7,9 +7,17 @@
 public class BuildValues {
     //Appended to the URL constructed in NativeAuthOAuth2Configuration, used for making calls to tenants on test slices
     @NonNull
-    public static final String DC = BuildConfig.DC;
+    public static String DC = BuildConfig.DC;
+    
+    public static void setDC(String dc) {
```

Comment:
I don't see this code in ADO

Reply:
I am testing out what is acceptable for Spotbugs.